media/css/base/banners/vpn-promo.scss

```diff
@@ -1,3 +1,7 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Doppelgaenger MPL

Reply:
Thanks. I was trying to look for those but I guess my eyes glazed over on this one. :)

Reply:
ditto!